<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageTeacherWidget</class>
 <widget class="QWidget" name="ManageTeacherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="insertButton">
     <property name="text">
      <string>添加</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>删除</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>取消</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>保存</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
